### DISCUSSION TITLE
Rails 7.1 Adds ActiveJob#perform_all_later To Enqueue Multiple Jobs At Once

### DISCUSSION THREAD

**Discussion by sonamvg** [therubyradar/submissions#192] [Submissions]
Bring efficiency and flexibility to job enqueuing in Rails 7.1 with ActiveJob's perform_all_later. Handle diverse job arguments, support bulk enqueuing, and offer enhanced control over job execution.

https://blog.saeloun.com/2023/11/27/rails-7-1-adds-active-job-perform-all-later/

**Comment by cjilbert504** [comment]
thanks @sonamvg!!!
